hceres/src/main/java/org/centrale/hceres/items/BelongsTeam.java

```diff
@@ -60,4 +60,5 @@ public class BelongsTeam implements Serializable {
     @JoinColumn(name = "team_id", referencedColumnName = "team_id", insertable = false, updatable = false)
     @ManyToOne
     private Team team;
+
```

Comment:
try to impact as less files as you can when commit, remove addition of white line